app/scripts/modules/core/src/pipeline/config/stages/manualJudgment/ManualJudgmentApproval.tsx

```diff
@@ -29,10 +34,36 @@ export class ManualJudgmentApproval extends React.Component<
       submitting: false,
       judgmentDecision: null,
       judgmentInput: {},
+      applicationRoles: {},
+      runOnce: true,
+      userRoles: [],
       error: false,
     };
   }
 
+  public componentDidMount() {
+      if (this.state.runOnce) {
+          const applicationName = this.props.execution.application;
+          ApplicationReader.getApplicationPermissions(applicationName)
+              .then(result => {
+                  if (typeof result !== 'undefined') {
```

Comment:
When will the result be `undefined`? When users don't have authz enabled?

Reply:
We need to avoid null pointer exceptions. This is a check to make sure that the result object is not undefined.